coursebuilder/modules/explorer/_static/js/course.js

```diff
@@ -8,10 +8,13 @@ $(document).ready(function documentReady() {
     margin: 10,
     nav: true,
     navText: [
-      '<i class="material-icons small">chevron_left</i>',
-      '<i class="material-icons small">chevron_right</i>'
+      '<i class="material-icons small" onclick="window.scrollTo(20, 20)">chevron_left</i>',
```

Comment:
@Sharonsyra this works. However, one can also navigate using the owl-dots hence callbacks may be more robust.

Reply:
@Sharonsyra Seen this? https://owlcarousel2.github.io/OwlCarousel2/docs/api-events.html

Reply:
Thank you. Working on it.